### ISSUE TITLE
Output in 64tass format produces null characters in .byte output

### ISSUE THREAD

**Issue by electrickery** [ice00/jc64#8]
The "Save in Tass64 format" file contains null characters instead of the expected "$00" output for the .byte format.

Using the jc64dis.jar version 2.9 on Linux with OpenJDK Runtime Environment (build 17.0.12+7-Debian-2deb12u1)

**Comment by ice00** [comment]
Can you share a .dis file with the problem (sent to iceteam @ altervista org if it is not to be public) as I'm not able to replicate the error with my test. Thanks.

**Comment by electrickery** [comment]
Hi ice00,

Thank you for the response. The issue might be related to another issue I had with a disassembly of a double ROM (CHIP) .CRT file. The disassembly didn't contain the second ROM contents which was present in the CRT file. 

So I removed the first ROM from the original .CRT and got a disassembly of the second ROM. I didn't correct the CRT header to a single ROM file, which is probably a mistake. Could this trigger the null characters? In the zip are the original CRT, the hacked single CHIP .CRT and the disassembly of the latter.

I couldn't open the disassembly file with my editor until I replaced the NULLS with $00.

[CRT2ndROM.zip](https://github.com/user-attachments/files/18247112/CRT2ndROM.zip)
